dev/generate-multi-command-encoder.sh

```diff
@@ -53,7 +54,7 @@ echo "extension ValkeyCommandEncoder {"
 # note:
 # - widening the interval below (eg. going from {1..15} to {1..25}) is Semver minor
 # - narrowing the interval below is SemVer _MAJOR_!
-for n in {1..15}; do
```

Comment:
I think, that we should do all the versions above 10 with a parameter pack implementation. But that can be a follow up.

Reply:
But the parameter pack version crashes the release build, otherwise we would use a parameter pack all the time